pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -630,7 +634,9 @@ func getDeviceMountPath(volume *reconstructedVolume) (string, error) {
 
 func (rc *reconciler) updateStates(volumesNeedUpdate map[v1.UniqueVolumeName]*reconstructedVolume) error {
 	// Get the node status to retrieve volume device path information.
-	rc.updateDevicePath(volumesNeedUpdate)
+	if err := rc.updateDevicePath(volumesNeedUpdate); err != nil {
+		return err
```

Comment:
@jingxu97 will returning error here cause reconstruction for working volume types like emptydir to fail?

Reply:
@jingxu97 
See the question above.

Reply:
right, I think we should not return error here. Just log in updateDevicePath() function.